youtube_dl/extractor/rtve.py

```diff
@@ -40,6 +40,28 @@ class RTVEALaCartaIE(InfoExtractor):
             'series': 'Balonmano',
         },
         'expected_warnings': ['Failed to download MPD manifest', 'Failed to download m3u8 information'],
+    }, {
+        'url': 'http://www.rtve.es/play/videos/balonmano/o-swiss-cup-masculina-final-espana-suecia/2491869/',
+        'md5': '1d49b7e1ca7a7502c56a4bf1b60f1b43',
+        'info_dict': {
+            'id': '2491869',
+            'ext': 'mp4',
+            'title': 'Balonmano - Swiss Cup masculina. Final: España-Suecia',
+            'duration': 5024.566,
+            'series': 'Balonmano',
+        },
+        'expected_warnings': ['Failed to download MPD manifest', 'Failed to download m3u8 information'],
+    }, {
+        'url': 'http://www.rtve.es/playz/videos/balonmano/o-swiss-cup-masculina-final-espana-suecia/2491869/',
```

Comment:
This URL redirects to https://www.rtve.es/play/videos/balonmano/o-swiss-cup-masculina-final-espana-suecia/2491869/

Reply:
I think we could remove this test because we don't need support for `playz`. What do you think?

Reply:
If you want yt-dl to handle URLs containing `...playz/...`, leave the pattern as `...playz?/...`. Any redirection happens after the URL gets matched.